finality-aleph/src/sync/service.rs

```diff
@@ -40,6 +41,8 @@ pub struct Service<
     chain_events: CE,
     justifications_from_user: mpsc::UnboundedReceiver<J::Unverified>,
     additional_justifications_from_user: mpsc::UnboundedReceiver<J::Unverified>,
+    _block_requests_from_user:
+        mpsc::UnboundedReceiver<<<J as Justification>::Header as Header>::Identifier>,
```

Comment:
type alias for this receiver? We should also maybe add such things to style guides :P

Reply:
Some alias would really be nice. <_<